x/sequencer/keeper/query_last_sort_key_test.go

```diff
@@ -0,0 +1,127 @@
+package keeper_test
```

Comment:
package keeper

Reply:
That's the convention in `CometBFT` and `CosmosSDK`. For consistency, I'm leaving it unchanged.